# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
float32 activity_thresh
float32 z_thresh
float32 mag_thresh
duration timeout
